tests/virtio_rng_test.go

```diff
@@ -0,0 +1,78 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+const (
+	DDCommand       =  "dd count=10 bs=1024 if=/dev/%s of=/tmp/%s.txt\n"
+	CheckRNGDevice  =  "cat /sys/devices/virtual/misc/hw_random/%s\n"
+	CheckFileSize   =  "ls /tmp/%s.txt | wc -l'\n"
+	outputDevice    =  "virtio_rng.0" 
+)
+var RngAvailable    string
```

Comment:
do you need to export these names ? everything with capital letter is visible outside of module. you should use it consciously.

Reply:
fixed it to private